---
title: "Mosquito Vectors"
excerpt: "Chromosome Painting in Anopheles funestus <br/><img src='/images/example.smoothed.stacked.jpg'>"
collection: portfolio
---
Next-generation sequencing (NGS) technology goes far beyond whole genome sequencing. For example, instead of sequencing multiple individual clones of a PCR amplicon using NGS we can now label each amplicon with a unique bar-code sequence and perform one sequencing experiment using new Illumina or PacBio technology. This allows for screening of many samples simultaneously while multiplexing multiple genetic amplicons (sometimes 100 of amplicons) all while saving money and labor.
	In 2015, I worked with a graduate student to publish four genomes of mosquitoes from the understudied Anopheles punctulatus species complex in Papua New Guinea (Logue et al. 2015). The project mainly encompassed genome assembly and population genetics in the context of mosquito biology. Our results demonstrated a dramatic difference in population histories among two common species: An. punctulatus s.s. and An. farauti no. 4. We hypothesized that the population size increase of An. punctulatus s.s. was due to its preference for human disturbed habitats in contrast to An farauti no. 4 which preferred larval habitats with higher water clarity and more canopy cover (Logue et al. 2015).
	During 2014, myself and Dr. David Serre were invited to Colombia to collaborate on an experiment stemming from work on NGS and mosquito blood meal analysis (Logue et al. 2016. doi: 10.1371/journal.pntd.0004512). In Colombia, we collected adult mosquitoes which were later analyzed using the methods of Logue 2016 for blood meal and host quantification. We then attempted a novel experiment by sampling both mosquito larvae and water from multiple different types of habitats. We used the eDNA methods outlined in Cannon et al. 2016 and Cannon et al. 2017, to classify both the macro and micro communities in each larval habitat. Using the collected mosquito larvae, we also compared the gene expression (by sequencing the transcriptome) with presence/absence of invertebrates and bacteria. This dataset was largely proof-of-concept, and we only found weak correlations with certain types of bacteria and presence/absence of mosquito species (mainly due to small sample sizes). 
