---
permalink: /
title: "About me"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

My education background was initially in ecology where I was studying conservation of water
and preservation of natural resources.After some years working in the field of ecology, I chose to
attended graduate school at the University of Georgia (UGA) to train in population genetics. It was at
this time that I took a course with then UGA faculty Dr. Pejman Rohani in modelling infectious
diseases. It was during this course that I saw an opportunity to use my background in ecology and
population genetics to work towards parameterizing models of infectious disease.
I joined the Center for Global Health & Diseases at Case Western Reserve University (CWRU)
in 2011 as a post-doctoral fellow. At CWRU I focused on applying population genetics to the study of lymphatic filariasis, specifically cases caused by the nematode Wuchereria bancrofti. During my time at CWRU we published the first population genomics study on W. bancrofti and, at the time, the only population genomics study of any filarial nematode. My research has continued to integreate ecology into hypotheses of genetic variation in populations of Nematodes and Mosquitoes. Recent work has invovled integrating eDNA into community ecology of vector species. 

Wuchereria bancrofti
======
Population genomics has only recently been integrated into studies of infectious diseases. This is because DNA from parasites can be hard to obtain and many epidemiologists are not trained in population genetics. In the last five years, I have strived to combined population genomics, epidemiology, and ecology to study the human parasite Wuchereria bancrofti. Wuchereria bancrofti (Wb) is a nematode causing Lymphatic filariasis, affecting over 120 million people in 73 countries with 1.39 billion people living in Wb endemic regions. 
In 2011, we started with only a single gene on the mitochondrial genome, COI, to study population structure in communities infected with Wb parasites (Small et al. 2013). We combined our estimates of migration and population structure with survey data collected from the same communities. We concluded that high connectivity between communities would make it difficult to eliminate Wb, unless all communities were treated simultaneously. We also identified a single mitochondrial lineage (denoted as a single unique haplotype) that was present in 95% of all Wb infections.
In 2015, after sequencing the whole mitochondrial genome of multiple Wb isolates (Ramesh et al. 2012), we used whole genome data from 15 individual Wb worms to publish the first population genomics study of Wb. This project used a larval stage of Wb that only occurs within mosquitoes; still we did not have enough DNA for traditional genome sequencing. We thus modified an available approach using Nextera chemistry (Illumina) and singe-cell whole genome amplification to perform genome sequencing (Small et al 2016).
From the genomic data, we made three novel insights into Wb life history. First, we confirmed a hypothesis that female Wb worms mate with multiple Wb male worms by utilizing a multiple-dimensional relationship statistics based on whole genome variation. We used this data to infer pedigree relationships between worms in our samples, allowing for the first time a non-invasive estimate of adult Wb worms in an infection. Second, we reconstructed the population history of Wb in Papua New Guinea. This included a hypothesis of migration rather than isolation in spreading Wb from SE Asia to Papua New Guinea and other Pacific Islands. Finally, we searched the Wb genomes for signs of natural selection against commonly used chemotherapeutic drugs. While we did not find any signs of emerging drug resistance, we did discover that a gene critical for Wb’s survival in the human host, macrophage-inhibitory-factor (MIF-1), was under directional selection. A previous experiment using Brugia malayi, which can be cultured in lab animals, demonstrated that MIF-1 proteins influence the movement of macrophages in vitro (Pastrana et al. 1998. Infection and Immunity. vol. 66 no. 12 5955-5963). Future studies of this gene and its natural genetic variation will aid in Wb control and inform on the larger immunological landscape of parasite and the human immune system.

Mosquito Vectors
======
Next-generation sequencing (NGS) technology goes far beyond whole genome sequencing. For example, instead of sequencing multiple individual clones of a PCR amplicon using NGS we can now label each amplicon with a unique bar-code sequence and perform one sequencing experiment using new Illumina or PacBio technology. This allows for screening of many samples simultaneously while multiplexing multiple genetic amplicons (sometimes 100 of amplicons) all while saving money and labor.
	In 2015, I worked with a graduate student to publish four genomes of mosquitoes from the understudied Anopheles punctulatus species complex in Papua New Guinea (Logue et al. 2015). The project mainly encompassed genome assembly and population genetics in the context of mosquito biology. Our results demonstrated a dramatic difference in population histories among two common species: An. punctulatus s.s. and An. farauti no. 4. We hypothesized that the population size increase of An. punctulatus s.s. was due to its preference for human disturbed habitats in contrast to An farauti no. 4 which preferred larval habitats with higher water clarity and more canopy cover (Logue et al. 2015).
	During 2014, myself and Dr. David Serre were invited to Colombia to collaborate on an experiment stemming from work on NGS and mosquito blood meal analysis (Logue et al. 2016. doi: 10.1371/journal.pntd.0004512). In Colombia, we collected adult mosquitoes which were later analyzed using the methods of Logue 2016 for blood meal and host quantification. We then attempted a novel experiment by sampling both mosquito larvae and water from multiple different types of habitats. We used the eDNA methods outlined in Cannon et al. 2016 and Cannon et al. 2017, to classify both the macro and micro communities in each larval habitat. Using the collected mosquito larvae, we also compared the gene expression (by sequencing the transcriptome) with presence/absence of invertebrates and bacteria. This dataset was largely proof-of-concept, and we only found weak correlations with certain types of bacteria and presence/absence of mosquito species (mainly due to small sample sizes). 

Biodiversity and Next-generation sequencing
======
My recent research has been mainly working on infectious diseases and vectors, but I still try to get out and do some solid field biology. In 2012, the David Serre lab took a trip down the Cuyahoga River in Ohio. The goal was to have some fun away from the lab bench, but the result was a study that used 50ml of river water and eDNA to explore the biodiversity of the Cuyahoga River Valley.
